Mathlib/SetTheory/Game/Ordinal.lean

```diff
@@ -180,4 +180,32 @@ termination_by a b => (a, b)
 theorem toPGame_add_mk' (a b : Ordinal) : (⟦a.toPGame⟧ + ⟦b.toPGame⟧ : Game) = ⟦(a ♯ b).toPGame⟧ :=
   Quot.sound (toPGame_add a b)
 
+/-- The product of ordinals as games corresponds to natural multiplication of ordinals. -/
+theorem toPGame_mul : ∀ a b : Ordinal.{u}, a.toPGame * b.toPGame ≈ (a ⨳ b).toPGame
```

Comment:
Can you turn it around and make it simp?

Reply:
This is an equivalence, not an equality. As for the subsequent result written for quotients, I'd argue that it should either stay in its current direction or not be a `simp` lemma at all. The intention of these theorems is to turn the more complex operations on games into comparatively well-characterized operations on ordinals.

Reply:
Actually, I think it makes sense to simply remove the `simp` lemma. Even if the RHS is simpler to work with, it's not structurally simpler in the way `simp` is supposed to be.

Reply:
> The intention of these theorems is to turn the more complex operations on games into comparatively well-characterized operations on ordinals.

Yes, but it is also a distributivity lemma, and distributivity lemmas are always written in the form "non-distributed = distributed"

Reply:
I'll rewrite the lemmas, but keep the simps out then.